File: Samples/Samples/NotebookSample.cs
using System;
using Xwt;
using Xwt.Drawing;

namespace Samples
{
	public class NotebookSample: VBox
	{
		public NotebookSample ()
		{
			Notebook nb = new Notebook ();
			nb.Add (new Label ("First tab content"), "First Tab");
			nb.Add (new MyWidget (), "Second Tab");
			PackStart (nb, BoxMode.FillAndExpand);
		}
	}
	
	class MyWidget: Canvas
	{
		protected override void OnDraw (Context ctx)
		{
			ctx.SetLineWidth (5);
			ctx.SetColor (new Color (1.0f, 0f, 0.5f));
			ctx.Rectangle (5, 5, 200, 100);
			ctx.FillPreserve ();
			ctx.SetColor (new Color (0f, 0f, 1f));
			ctx.Stroke ();
		}
	}
}

File: SamplesVB/Samples/NotebookSample.vb
Imports System
Imports Xwt
Imports Xwt.Drawing

Namespace Samples
    Public Class NotebookSample
        Inherits VBox

        Public Sub New()
            Dim nb As Notebook = New Notebook()
            nb.Add(New Label("First tab content"), "First Tab")
            nb.Add(New MyWidget(), "Second Tab")
            MyBase.PackStart(nb, BoxMode.FillAndExpand)
        End Sub
    End Class

    Friend Class MyWidget
        Inherits Canvas

        Protected Overrides Sub OnDraw(ctx As Context, dirtyRect As Rectangle)
            ctx.SetLineWidth(5.0)
            ctx.SetColor(New Color(1.0, 0.0, 0.5))
            ctx.Rectangle(5.0, 5.0, 200.0, 100.0)
            ctx.FillPreserve()
            ctx.SetColor(New Color(0.0, 0.0, 1.0))
            ctx.Stroke()
        End Sub
    End Class

End Namespace
